packages/mobile/src/components/drawers/ArtistPickConfirmationDrawer.tsx

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useMemo } from 'react'
+
+import { useGetCurrentUser } from '@audius/common/api'
+import { tracksSocialActions, useArtistPickModal } from '@audius/common/store'
+import { useDispatch } from 'react-redux'
+
+import { ConfirmationDrawer } from './ConfirmationDrawer'
+
+const messages = {
+  add: {
+    header: 'Set your Artist Pick',
+    description:
+      'This track will appear at the top of your profile, above your recent uploads, until you change or remove it.',
+    confirm: 'Set Track'
+  },
+  update: {
+    header: 'Change your artist pick?',
```

Comment:
should this header be sentence case like the others?

Reply:
also is the question mark needed?

Reply:
yes great callout. technically i think drawer title is cap-case but should be consistent